script/env/install_env.sh

```diff
@@ -59,3 +59,6 @@ deployUser=${deployUser:-"dolphinscheduler"}
 
 # The root of zookeeper, for now DolphinScheduler default registry server is zookeeper.
 zkRoot=${zkRoot:-"/dolphinscheduler"}
+
+#whether the installation user has sudo permission
+sudoEnable=${sudoEnable:-"true"}
```

Comment:
It's better to add this to the document.

Reply:
Considering the code style, It would be better to use `# whether` instead of `#whether`